README.md

```diff
@@ -0,0 +1,4 @@
+# pomodoro-clock-angular
```

Comment:
Nice that you are using font-awesome icons, for a different project try using google material icons to familiarize yourself with them.